modules/javafx.web/src/main/native/Source/JavaScriptCore/b3/B3DuplicateTails.cpp

```diff
@@ -77,6 +77,15 @@ class DuplicateTails {
             if (block->last()->type() != Void) // Demoting doesn't handle terminals with values.
                 continue;
 
+            bool canCopyBlock = true;
+            for (Value* value : *block) {
+                if (value->kind().isCloningForbidden()) {
+                    canCopyBlock = false;
+                    break;
+                }
+            }
+
+            if (canCopyBlock)
             candidates.add(block);
```

Comment:
The indentation looks off here.

Reply:
I'll reapprove if you choose to fix this.

Reply:
Indentation corrected